terraform/variables.tf

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 Google LLC
+Copyright 2019 Google LLC 
```

Comment:
make lint is failing because there is a whitespace at the end of this line

Reply:
Done. Pushed the change